File: app/src/main/java/com/boardgamegeek/util/fabric/SortEvent.java
package com.boardgamegeek.util.fabric;

import com.crashlytics.android.answers.Answers;
import com.crashlytics.android.answers.CustomEvent;

public class SortEvent extends CustomEvent {
	public SortEvent() {
		super("Sort");
	}

	public static void log(String contentType, String sortBy) {
		SortEvent event = newInstance(contentType);
		event.putCustomAttribute("SortBy", sortBy);
		Answers.getInstance().logCustom(event);
	}

	private static SortEvent newInstance(String contentType) {
		SortEvent event = new SortEvent();
		event.putCustomAttribute("contentType", contentType);
		return event;
	}

}

File: app/src/main/java/com/boardgamegeek/util/fabric/SortEvent.kt
package com.boardgamegeek.util.fabric

import com.crashlytics.android.answers.Answers
import com.crashlytics.android.answers.CustomEvent

class SortEvent : CustomEvent("Sort") {
    companion object {
        @JvmStatic
        fun log(contentType: String, sortBy: String) {
            val event = SortEvent().apply {
                putCustomAttribute("contentType", contentType)
                putCustomAttribute("SortBy", sortBy)
            }
            Answers.getInstance().logCustom(event)
        }
    }
}
